services/src/main/java/org/fao/geonet/api/users/RegisterApi.java

```diff
@@ -80,7 +80,8 @@
 
     @ApiOperation(value = "Create user account",
         nickname = "registerUser",
-        notes = "User is created with a registered user profile. Password is sent by email. Catalog administrator is also notified.")
+        notes = "User is created with a registered user profile. username field is ignored and the email is used as " +
```

Comment:
I am not against this, but... this is a change for a different thing, right?

Reply:
Thisis only a clarification in the documentation I found not being accurate while doing the implementation of these changes, but yes, not related.